node/src/BaseClient.ts

```diff
@@ -1358,6 +1359,51 @@ export class BaseClient {
         return this.createWritePromise(createHGet(key, field));
     }
 
+    /**
+     * Iterates incrementally over a hash.
+     *
+     * See https://valkey.io/commands/hscan for more details.
+     *
+     * @param key - The key of the set.
+     * @param cursor - The cursor that points to the next iteration of results. A value of `"0"` indicates the start of the search.
+     * @param options - The {@link BaseScanOptions}.
+     * @returns An array of the `cursor` and the subset of the hash held by `key`.
```

Comment:
Maybe we can keep only the first phrase and add "see example for more details"? I did this in some commands.

Reply:
i think we should decide this as a team if we want to follow this convention, but i also don't mind shortening it down